README.md

```diff
@@ -5,14 +5,15 @@ QJSLib is a JavaScript utility library that provides functionality for QRadar ap
 * [CONTRIBUTING](CONTRIBUTING.md)
 * [MAINTAINERS](MAINTAINERS.md)
 * [CHANGELOG](CHANGELOG.md)
+* [DOCUMENTATION](https://github.com/IBM/qjslib/wiki/qappfw)
 
 ## How to use
 ### Browser vs module
-The build process for this project generates two seperate *qappfw.js* files, one in *build/* and one in *lib/*. 
+The build process for this project generates two seperate files in *lib/*.
 
-The file in *lib/* is not minified and designed to be bundled with your application code.
+The file in *qappfw.js* is not minified and designed to be bundled with your application code.
 
-The file in *build/* contains a number of polyfills and is minified to ensure compatability with older browsers, and as such is suitable for direct use from the browser. 
+The file in *qappfw.min.js* contains a number of polyfills and is minified to ensure compatability with older browsers, and as such is suitable for direct use from the browser. 
```

Comment:
Also weird

Reply:
*qappfw.min.js* is minified with polyfills for browser compatibility, it is designed to be used directly in the browser.

How about this?

Reply:
...compatibility. It is designed....